backend/uclapi/.env.example

```diff
@@ -64,9 +64,13 @@ OPBEAT_ORG_ID=
 OPBEAT_APP_ID=
 OPBEAT_SECRET_TOKEN=
 
-# Keen
+# Keen settings
 KEEN_PROJECT_ID=
 KEEN_WRITE_KEY=
 
 # Sentry settings
 SENTRY_DSN=
+
+# Redis settings
+REDIS_UCLAPI_HOST=
+REDIS_UCLAPI_CLIENT_CLASS=
```

Comment:
Could you explain what the client_class variable does, and maybe include an example in the .env.example?